x-pack/plugins/secops/public/components/page/network/network_top_n_flow_table/select_type.tsx

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiSuperSelect } from '@elastic/eui';
+import React from 'react';
+import { pure } from 'recompose';
+
+import { NetworkTopNFlowDirection, NetworkTopNFlowType } from '../../../../graphql/types';
+
+const toggleTypeOptions = [
+  {
+    value: NetworkTopNFlowType.source,
+    inputDisplay: 'By Source IP',
```

Comment:
These will need to be i18n'd. Not sure if you can simply join the other translations with `By` or if you'll need to make separate keys for each of these though.